playbook/app/pb_kits/playbook/pb_checkbox/checkbox.html.erb

```diff
@@ -6,12 +6,12 @@
                 ) do %>
   <%= content.presence || object.input %>
   <% if object.indeterminate %>
-    <span class="pb_checkbox_indeterminate">
+    <span data-pb-checkbox-icon-span="true" class="pb_checkbox_indeterminate">
       <%= pb_rails("icon", props: { icon: "minus", classname: "indeterminate_icon", fixed_width: true}) %>
       <%= pb_rails("icon", props: { icon: "check", classname: "check_icon hidden", fixed_width: true}) %>
     </span>
   <% else %>
-    <span class="pb_checkbox_checkmark">
+    <span data-pb-checkbox-icon-span="true" class="pb_checkbox_checkmark">
```

Comment:
Doc example in milano looks and works great! I wonder if the addition of a data class here and the removal of `checked_html` class in checkbox.rb is enough of a structural kit change to warrant alpha testing just to confirm pre-existing checkbox functionality not altered.

Reply:
I would say, lets still alpha test this so when we do the RC, it requires no testing and can just go through!